### PR TITLE
Use 'sn.exe' package from dotnet-core feed

### PR DESCRIPTION
**Pull request by JohnTortugo**

### CODE REVIEW

File: src/Microsoft.DotNet.Arcade.Sdk/tools/DefaultVersions.props [hunk 1]
```diff
@@ -72,6 +72,7 @@
     <MicrosoftSourceLinkVersion Condition="'$(MicrosoftSourceLinkVersion)' == ''">1.0.0-beta-63401-01</MicrosoftSourceLinkVersion>
     <MicrosoftSymbolUploaderBuildTaskVersion Condition="'$(MicrosoftSymbolUploaderBuildTaskVersion)' == ''">1.0.0-beta-63008-01</MicrosoftSymbolUploaderBuildTaskVersion>
     <VSWhereVersion Condition="'$(VSWhereVersion)' == ''">2.5.2</VSWhereVersion>
+    <VSWhereVersion Condition="'$(SNVersion)' == ''">1.0.0</VSWhereVersion>
```
Comment (jcagme) [inline src/Microsoft.DotNet.Arcade.Sdk/tools/DefaultVersions.props]: >VSWhereVersion [](start = 5, length = 14)

Same name from above

File: src/Microsoft.DotNet.Arcade.Sdk/tools/Tools.proj [hunk 2]
```diff
@@ -39,6 +39,9 @@
     </RestoreSources>
   </PropertyGroup>
   <ItemGroup>
+    <!-- Copy of 'sn.exe' in form of NuGet package. -->
+    <PackageReference Include="sn" Version="$(SNVersion)" IsImplicitlyDefined="true" />
```
Comment (jcagme) [inline src/Microsoft.DotNet.Arcade.Sdk/tools/Tools.proj]: ><P [](start = 4, length = 2)

Nit: remote empty line bellow

**Comment by JohnTortugo** [comment]
I'm trying to get an official build of this branch to get a new version of Arcade SDK and then use that new version in another build to validate that the SDK is working. I'm hitting some "feed locked" timeouts but I think everything we'll be done during the afternoon.

**Comment by JohnTortugo** [comment]
This branch build correctly here: https://dnceng.visualstudio.com/internal/_build/results?buildId=43101&view=logs

Testing the published package here: https://dnceng.visualstudio.com/internal/_build/results?buildId=43127&view=logs

**Comment by JohnTortugo** [comment]
Both builds succeeded now.

**Pull request merged**